src/components/Account/Account.js

```diff
@@ -37,17 +37,17 @@ export const Account = (props) => {
       <div>
         <div className={classes.primaryContainer}>
           {primary ||
-           <div onClick={onEditClick} className={classes.accountName}>
-             <div>{ name }</div>
-             {editable && <div style={{ marginLeft: '5px' }}><EditIcon style={styles.editIcon} /></div>}
-           </div>}
+          <div onClick={onEditClick} className={classes.accountName}>
+            <div>{ name }</div>
+            {editable && <div style={{ marginLeft: '5px' }}><EditIcon style={styles.editIcon} /></div>}
```

Comment:
should use withStyles here, which will make this no longer inline, and use classNames instead.